symfony.lock

```diff
@@ -328,9 +325,6 @@
     "phpdocumentor/type-resolver": {
         "version": "1.4.0"
     },
-    "phpseclib/phpseclib": {
-        "version": "3.0.9"
-    },
```

Comment:
How did this get removed

Reply:
You asked me to remove activitypub. This got removed automatically as well with activitypub ..

Reply:
Follow-up PR now: https://github.com/MbinOrg/mbin/pull/1135